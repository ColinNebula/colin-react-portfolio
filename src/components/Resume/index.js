import React from 'react';


function Resume() {

    return (
        <section className="my-5" >
            <h1 id="resume">My Web Development Resume</h1>

            <div className="my-2">
                <h2> 
                Colin Nebula
                
                </h2>
                <h3>Full Stack Developer</h3>
                <p>
                   HTML | CSS | JavaScript | MYSQL | NodeJs | NoSQL | React | MERN | Progressive Web Applications (PWA) | Model-View-Controller (MVC) | IndexedDB
                </p>
                
            </div>
        </section>
    )
}

export default Resume;