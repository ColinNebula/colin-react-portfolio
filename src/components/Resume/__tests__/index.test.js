import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Resume from '..';
// Component being testedafterEach(cleanup);

describe('Resume component', () => {
    it('renders', () => {
      render(<Resume />);
    });
    
    it('matches snapshot DOM node structure', () => {
      const { asFragment } = render(<Resume />);
      
      expect(asFragment()).toMatchSnapshot();
    });
  })