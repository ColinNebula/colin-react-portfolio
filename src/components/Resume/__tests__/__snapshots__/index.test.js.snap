// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <h1
      id="resume"
    >
      My Web Development Resume
    </h1>
    <div
      class="my-2"
    >
      <h2>
        Colin Nebula
      </h2>
      <h3>
        Full Stack Developer
      </h3>
      <p>
        HTML | CSS | JavaScript | MYSQL | NodeJs | NoSQL | React | MERN | Progressive Web Applications (PWA) | Model-View-Controller (MVC) | IndexedDB
      </p>
    </div>
  </section>
</DocumentFragment>
`;
